File: ex06/ex06_05/ex06_05.py
from src.leap import *

def main():
  while True:
    year = int(input("Enter year: "))
    if year < 0:
      print("Invalid year. Exiting.")
      break
    if is_leap_year(year):
      print(f"Year {year} is a leap year.")
    else:
      print(f"Year {year} is a common year.")

if __name__ == "__main__":
  main()

File: ex06/ex06_05/ex06_05.js
import promptSync from 'prompt-sync';
const prompt = promptSync();
import { isLeapYear } from './src/leap.js';

function main() {
  let year = 0;

  do {
    year = prompt("Enter year: ");
    if (year < 0) {
      console.log("Invalid year. Exiting.");
      break;
    }
    if (isLeapYear(year)) {
      console.log(`Year ${year} is a leap year.`);
    } else {
      console.log(`Year ${year} is a common year.`);
    }
  } while (true);
  return 0;
}
main();